edweiss-app/model/school/courses.ts

```diff
@@ -8,15 +8,6 @@ export type CourseID = string & {};
 export type Credits = number & {};
 
 export type Section = "IN" | "COM" | "PH" | "MT";
```

Comment:
Why did you remove the course colors?

Reply:
Hello,
The model files should have nothing to do with colors and display, I moved `courseColors` to the, I believe, more appropriate `constants/Colors.ts` file of the app.